### PR TITLE
Implemented Device Instructions Screen

### LINKED ISSUE #349: Device Connection Instructions Screen
**Actual Behaviour.**  
Missing Device Connection Instructions Screen.

**Expected Behaviour**  
Device Connection Instructions Screen on clicking device connection or toolbar icon.

**Steps to reproduce it**  
Click on device connection on nav drawer or icon.

**Additional context**  
> Not available

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #349 

**Changes**: Implemented Device Instructions Screen

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3366975/app-debug.zip)

### CODE REVIEW

**Comment by jddeep** [comment]
@CloudyPadmal @mariobehling Please can we get the previous PRs #345 and #346 merged ASAP (Suggested changes were made to those). Otherwise, build failures will rise and to fix that, I need to push similar changes to new PRs as of previous PRs.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -75,10 +75,18 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
     <string name="other_head_nav_drawer">Other</string>
     <string name="share_nav_drawer">Share</string>
     <string name="send_nav_drawer">Send</string>
     <string name="device_connected">Device is Connected</string>
+    <string name="steps_header">Steps to Connect Device:</string>
+    <string name="number_one">1.</string>
+    <string name="number_two">2.</string>
+    <string name="number_three">3.</string>
+    <string name="step_one_info">Connect a Micro USB to NeuroLab</string>
+    <string name="step_two_info">Connect the other end of Micro USB to a OTG</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="step_two_info">Connect the other end of Micro USB to an OTG cable</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -75,10 +75,18 @@ A difference to existing projects like OpenBCI is that it will not be necessary
...
+    <string name="step_three_info">Connect the OTG to the phone</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="step_three_info">Connect the OTG cable to the phone</string>
```

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -107,6 +115,10 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="freq_label">Frequency (Hertz)</string>
     <string name="eeg_data_label">EEG data (mV)</string>
     <string name="play_graph">Play Graph</string>
+    <string name="save_focus_data">Save Focus Data</string>
+    <string name="focus_program_info">This Program Mode helps you increase you increase your focus and concentration levels.</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Check this line

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by mariobehling** [comment]
Thanks, please also resolve conflicts.

**Comment by CloudyPadmal** [comment]
Conflicts again.

**Comment by jddeep** [comment]
This can be merged now. Thanks.

**Pull request merged**